pkg/types/cluster_metadata.go

```diff
@@ -24,5 +24,6 @@ type ClusterMetadata struct {
 	SSHAgentHostname    string
 	SSHAgentPort        string
 	Certificate         []byte
+	CliVersionRange     []string
```

Comment:
What happens if a backend doesn't return the CliversionRange? I think the CLI will fail in that case